keps/sig-api-machinery/4006-transition-spdy-to-websockets/README.md

```diff
@@ -754,6 +736,12 @@ in back-to-back releases.
 
 #### GA
 
+- `kubectl` environment variables and API Server feature gates are locked to on by default.
+- Deprecate `kubectl` environment variables and API Server feature gates for future removal.
+- Force synthetic RBAC `CREATE` authorization check for WebSocket upgrades on the following
+  subresources: `pods/exec`, `pods/attach`, and `pods/portforward`. This additional check
+  will be gated by the API Server `ForceRBACCreateCheck` feature flag, which defaults to
```

Comment:
I'd name this as specifically as possible, to indicate what it does... maybe something like `AuthorizePodWebsocketUpgradeCreatePermission`

Reply:
I believe I have addressed this. Please let me know what you think.